stack/debug.rst

```diff
@@ -43,7 +43,7 @@ Then you can write your task to optionally enable a display (or whatever) by doi
 Task Users
 ==========
 
-Refer to the `task documentation <http://doxygen.lsst.codes/stack/doxygen/x_mainDoxyDoc/group___l_s_s_t__task__documentation.html>`_ and look for “debug variables” to discover what debugging options are available for existing ``Task``\s.
+Refer to the `task documentation <https://pipelines.lsst.io/v/daily/modules/lsstDebug/index.html>`_ and look for "debugging" to discover what debugging options are available for existing ``Task``\s.
```

Comment:
Oh, but one thing: this doesn't look exactly equivalent.  I don't see a "debugging" section in the lsstDebug page, for example.

Reply:
Individual tasks do (sometimes? rarely?) have "Debugging" sections: https://pipelines.lsst.io/v/weekly/modules/lsst.meas.algorithms/tasks/lsst.meas.algorithms.SourceDetectionTask.html#debugging
I think you want to point to https://pipelines.lsst.io/v/daily/tasks.html instead of the original doxygen page.